lib/chef/policy_builder/policyfile.rb

```diff
@@ -137,6 +147,7 @@ def build_node
         Chef::Log.info("Run List expands to [#{run_list_with_versions_for_display.join(', ')}]")
 
         events.node_load_completed(node, run_list_with_versions_for_display, Chef::Config)
+        events.run_list_expanded(run_list_expansion_ish.to_hash_for_json)
```

Comment:
The consumers of this data are expecting a `RunListExpansion`-compatible object here, like in https://github.com/chef/chef/blob/5c4dbb8b34f55e2e253f988cc958c8b0e78ccea5/lib/chef/policy_builder/expand_node_object.rb#L136